README.md

```diff
@@ -77,20 +77,27 @@ $ git clone https://github.com/saadpasta/developerFolio.git
 # Go into the repository
 $ cd developerFolio
 
+# Setup environment variables
+
+# For Linux
+$ cp env.example .env
+# For Windows
+$ copy env.example .env
```

Comment:
the `$` here seems to misguide people, where people would just put the `$` in the terminal and get errors.

Reply:
Good point, I'll remove them from all commands